docs/.vitepress/dark-mode.js

```diff
@@ -1,5 +1,5 @@
 ;(() => {
```

Comment:
https://vueuse.org/core/useDark/

Maybe we can use `useDark` from `vueuse`

Reply:
I had use `useDark` from `vueuse`.

But this is needed, because it init mode before script.

For example: https://github.com/antfu/vitesse/blob/5956b053f1c2025ba5bc9493e6c6984e4daa1b22/index.html#L12-L17